docs/readthedocs/source/doc/PythonAPI/LLM/optimize.rst

```diff
@@ -1,10 +1,24 @@
 BigDL-LLM PyTorch API
 =====================
 
-optimize model
+Optimize Model
 ----------------------------------------
 
-.. automodule:: bigdl.llm.optimize
+You can run any PyTorch model with one-line ``optimize_model`` to optimize, regardless of the library or API you are using.
+
+.. automodule:: bigdl.llm
     :members: optimize_model
     :undoc-members:
     :show-inheritance:
+
+
+
+Load Optimized Model
+----------------------------------------
+
+To save model space and speedup loading processes, you can save the model after low-bit optimization and load the saved low-bit model. Saving and loading operations can across various machines, regardless of their operating systems.
+
+.. automodule:: bigdl.llm.optimize
+    :members: load_low_bit
+    :undoc-members:
+    :show-inheritance:
```

Comment:
Should we also add API for `save_low_bit`?

Reply:
> Should we also add API for `save_low_bit`?

`save_low_bit` is a dynamical method of the optimized model, the API doc of this method may not be available based on current implementation.